### PR TITLE
NXP-27849: move nuxeo-document-thumbnail to Nuxeo Elements

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

File: i18n/messages.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
   "activity.workflowTaskAssigned": "started a review",
   "activity.workflowTaskCompleted": "completed a review task",
   "accessibility.logo": "Application Logo",
-  "accessibility.thumbnail": "Thumbnail of {0}",
```
Comment (guirenard) [inline i18n/messages.json]: Should we consider moving all locale labels as we have them all rather than going through the Crowdin flow again?

Comment (semisse) [reply i18n/messages.json]: Apparently Crowdin will override those changes. However I will translate those in the Crowdin platform itself.

**Comment by semisse** [comment]
Pipeline ok: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-sfialho/81/

**Comment by semisse** [comment]
Pipeline ok: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-sfialho/82/

**Pull request merged**